src/data/faq_de.json

```diff
@@ -1138,7 +1138,7 @@
                                 "anchor": "vac_cert_red_warning",
                                 "active": false,
                                 "textblock": [
-                                    "Sollte die Corona-Warn-App Ihnen eine Begegnung mit erhöhtem Risiko in einer roten Kachel anzeigen, begeben Sie sich, wenn möglich, in eine freiwillige Selbstquarantäne, bis Sie ein negatives Testergebnis vorweisen können. Dies gilt auch, wenn Sie geimpft oder genesen sind und keine Symptome zeigen. Bei einer Warnung über ein erhöhtes Risiko (Rote Kachel) haben Sie Anspruch auf einen kostenlosen Test. Dies muss nicht notwendigerweise ein PCR-Test sein.",
+                                    "Sollte die Corona-Warn-App Ihnen eine Begegnung mit erhöhtem Risiko in einer roten Kachel anzeigen, begeben Sie sich, wenn möglich, in eine freiwillige Selbstquarantäne, bis Sie ein negatives Testergebnis vorweisen können. Dies gilt auch, wenn Sie geimpft oder genesen sind und keine Symptome zeigen. Nach <a href='https://www.gesetze-im-internet.de/coronatestv_2021-10/__4a.html' target='_blank' rel='noopener'>§4a Testverordnung</a> haben Sie Anspruch auf einen kostenlosen Schnelltest. Weitere Infomrationen dazu finden Sie hier: <a href='https://www.bundesgesundheitsministerium.de/coronavirus/nationale-teststrategie/faq-covid-19-tests.html#c23926' target='_blank' rel='noopener'>Fragen und Antworten zu COVID-19 Tests (BMG)</a>.",

```

Comment:
"Infomrationen" is a typo

Reply:
Typo has been fixed in [`070ebd2`](https://github.com/corona-warn-app/cwa-website/pull/2921/commits/070ebd26220fa47e63bc42bee9cd03452232076d). Thanks for spotting! Please re-review.